src/Sylius/Behat/Context/Api/Shop/ProductContext.php

```diff
@@ -78,12 +81,32 @@ public function iOpenProductPage(ProductInterface $product): void
 
     /**
      * @When I browse products from taxon :taxon
+     * @When I browse products
      */
-    public function iBrowseProductsFromTaxon(TaxonInterface $taxon): void
+    public function iBrowseProductsFromTaxon(?TaxonInterface $taxon = null): void
     {
         $this->client->index();
-        $this->client->addFilter('taxon', $this->iriConverter->getIriFromItem($taxon));
-        $this->client->filter();
+
+        if ($taxon !== null) {
+            $this->client->addFilter('taxon', $this->iriConverter->getIriFromItem($taxon));
+            $this->client->filter();
+        }
+    }
+
+    /**
+     * @When I sort products by the lowest price first
+     */
+    public function iSortProductsByTheLowestPriceFirst(string $sortType = 'ascending'): void
```

Comment:
```suggestion
    public function iSortProductsByTheLowestPriceFirst(): void
```
do we need this argument? same in line 107